### PR TITLE
Joomla 4.1.1 & 3.10.7 (better the underlying framework packages)

### PR DESCRIPTION
**Pull request by zero-24**

### CODE REVIEW

File: joomla/filesystem/CVE-2022-23794.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+title:     Path Disclosure within joomla/filesystem class
+link:      https://developer.joomla.org/security-centre/871-20220302-core-path-disclosure-within-filesystem-error-messages.html
+cve:       CVE-2022-23794
+branches:
+    master:
+        time:     2022-03-29 18:00:00
+        versions: ['<1.6.2', '<2.0.1']
```
Comment (stof) [inline joomla/filesystem/CVE-2022-23794.yaml:8]: this is the wrong way to specify that it is patched in 2 different versions. You need 2 branches for that, to be able to mark `<1.6.2` and `>=2.0.0, <2.0.1` as affected.

**Review by stof — CHANGES_REQUESTED** [review]

File: joomla/filesystem/CVE-2022-23794.yaml (hunk 1 above)
Comment (zero-24) [reply joomla/filesystem/CVE-2022-23794.yaml:8]: Ah ok Nowdays we have 2.x-dev and 1.x-dev and no master any more. Will take a look and update the PR later. Thanks!

**Comment by zero-24** [comment]
I'm sorry I think i got it now correct can you please confirm @stof ?

**Review by Ocramius — APPROVED** [review]

**Review by stof — APPROVED** [review]

**Comment by zero-24** [comment]
Thanks 👍

**Pull request merged**